src/pages/TeamSettings/Service-Accounts-Table.vue

```diff
@@ -251,7 +253,8 @@ export default {
             name: key.name,
             created_at: key.created,
             expires: key.expires_at,
-            user_id: key.user_id
+            user_id: key.user_id,
+            created_by: this.isCloud ? key.created_by.username : ''
```

Comment:
Again, this should always be cloud.

Reply:
Again we can be confident that this page is all cloud so can simply remove the check for isCloud. 

**For info** - Actually here you'd need to make sure you import the isCloud getter - without it this is returning false and so not showing the created_by when it is present.   the 'isCloud' getter looks like this ```...mapGetters('api', ['isCloud'])``` in the 'computed' section of a component and checks the api vuex store.